### DISCUSSION TITLE
Is this project support newest vmware(17)?

### DISCUSSION THREAD

**Discussion by Squirre17** [paolo-projects/auto-unlocker#107] [Q&A]
as title

**Comment by QuickOK** [comment]
In my case VMware Player 17.6.0 (latest).

**Comment by zakaus** [comment]
Hi mate, 
I'm encountering the following error with VMware Workstation Pro 17.6.3 (the latest version).
**error in curl get request : couldn't resolve host name**
![image](https://github.com/user-attachments/assets/44729cfe-cad6-404d-9600-565626b71173)

**Reply by alexvedi** [reply]
Same error here :(

**Reply by korybantes** [reply]
same

**Reply by ujaann** [reply]
same
